src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
```

Comment:
I don't know where i can write is, so write there.
https://mate.academy/learn/java-core/java-github-tasks-tutorial#/video/1028
Rewatch this instruction, on 3:20 Bogdan recommended create different branch with solving task. Somewhere i see requirement about this, can't find now. Now you working in master branch. Always create for your homework different branch.

Reply:
Thank you, I always forget about it